guide/src/configuring_polling_policies.md

```diff
@@ -16,4 +16,223 @@ limitations under the License.
 
 # Configuring Polling Policies
 
-<!-- TODO(#574) - this is a placeholder to satisfy links -->
+The Google Cloud Client Libraries for Rust provide helper functions to simplify
+waiting and monitoring the progress of
+[LROs (Long-Running Operations)](/working_with_long_running_operations.md).
+These helpers use policies to configure the polling frequency and to determine
+what polling errors are transient and may be ignored until the next polling
+event.
+
+This guide will walk you through the configuration of these policies for all the
+long-running operations started by a client, or just for one specific request.
+
+There are two different policies controlling the behavior of the LRO loops:
+
+- The polling backoff policy controls how long the loop waits before polling
+  the status of a LRO that is still in progress.
+- The polling policy controls what to do on an polling error. Some polling
+  errors are unrecoverable, and indicate that the operation was aborted or the
+  caller has no permissions to check the status of the LRO. Other polling errors
+  are transient, and indicate a temporary problem in the client network or the
+  service.
+
+Each one of these policies can be set independently, and each one can be set for
+all the LROs started on a client or changed for just one request.
+
+## Prerequisites
+
+The guide uses the [Speech-To-Text V2] service to keep the code snippets
+concrete. The same ideas work for any other service using LROs.
+
+We recommend you first follow one of the service guides, such as
+[Transcribe speech to text by using the command line]. These guides will cover
+critical topics such as ensuring your project has the API enabled, your account
+has the right permissions, and how to set up billing for your project (if
+needed). Skipping the service guides may result in problems that are hard to
+diagnose.
+
+## Dependencies
+
+As it is usual with rust, you must declare the dependency in your
```

Comment:
nit: Rust should be capitalized. This happens in other guides too.

Reply:
Fixed here, and any other cases I found (still may have missed something)